service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/IotHubConnectionString.java

```diff
@@ -632,7 +647,19 @@ public URL getUrlCreateExportImportJob() throws MalformedURLException
         stringBuilder.append(URL_SEPARATOR_0);
         stringBuilder.append("create");
         stringBuilder.append(URL_SEPARATOR_1);
-        stringBuilder.append(URL_API_VERSION);
+        // The new api-version is only available in a few initial regions
+        // Control access via an environment variable. If a user wishes to try it out,
+        // they can set "EnabledStorageIdentity" to "1". Otherwise, the SDK will still
+        // default to the latest, broadly-supported api-version used in this SDK.
+        IotHubConnectionString iotHubConnectionString = new IotHubConnectionString();
+        if (iotHubConnectionString.IsStorageIdentityEnabled)
```

Comment:
Does this set the new api-version globally, or just for import/export devices job?

Reply:
Thanks for catching this. It is setting only for import export currently, will change it to all.

Reply:
No, we want it only for import/export job. Not all. Sorry for the confusion.

Reply:
As discussed offline, this is only required for import export which is what it is currently. No changes required, resolving